src/main/java/uk/gov/crowncommercial/dts/scale/cat/model/EventType.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.crowncommercial.dts.scale.cat.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * EventType
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EventType {
```

Comment:
Was there a reason to switch away from using the generated `DefineEventType` enum?  We've tried to use the generated classes so we have (some level of..) confidence that the API we're exposing matches the spec - this class introduces properties that are not part of the API's response type (see https://github.com/Crown-Commercial-Service/ccs-scale-api-definitions/blob/master/cat/CaT-service.yaml#L2971 for the correct type)